test/e2e-v2/cases/so11y/so11y-cases.yaml

```diff
@@ -30,7 +30,7 @@
       expected: expected/metrics-has-value.yml
     - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=meter_oap_instance_trace_count --instance-name=http://localhost:1234 --service-name=oap::oap-server |yq e 'to_entries' -
       expected: expected/metrics-has-value.yml
-    - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=meter_oap_instance_metrics_first_aggregation --instance-name=http://localhost:1234 --service-name=oap::oap-server |yq e 'to_entries' -
+    - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=meter_oap_instance_metrics_aggregation --instance-name=http://localhost:1234 --service-name=oap::oap-server |yq e 'to_entries' -
```

Comment:
You are changing `single value` -> `labeled value`, I assume the expected file should be updated too.

Reply:
> You are changing `single value` -> `labeled value`, I assume the expected file should be updated too.

I am not familiar  with `e2e-solly`. Take a moment, I will look at it. Thanks for your reminder and patience.

Reply:
You could copy this expected file into the so11y folder, https://github.com/apache/skywalking/blob/master/test/e2e-v2/cases/vm/expected/metrics-has-value-label.yml

Reply:
> You could copy this expected file into the so11y folder, https://github.com/apache/skywalking/blob/master/test/e2e-v2/cases/vm/expected/metrics-has-value-label.yml

ok

Reply:
@wu-sheng I have run the "swctl" command in my own computer. please help me review again. Thanks!